Signal/src/call/CallService.swift

```diff
@@ -81,6 +81,18 @@ enum CallError: Error {
 // Should be roughly synced with Android client for consistency
 fileprivate let connectingTimeoutSeconds = 120
 
+// Example: OWSProdError("blah", #file, #function, #line)
+func OWSProdError(_ __eventName: String, file: String, function: String, line: Int32) {
```

Comment:
Yeah, you're right.  I was trying to be consistent with the preprocessor macros, but it just doesn't make sense.